pkg/volume/hostpath/host_path_test.go

```diff
@@ -196,7 +191,9 @@ func TestProvisioner(t *testing.T) {
 		t.Errorf("Expected reclaim policy %+v but got %+v", v1.PersistentVolumeReclaimDelete, pv.Spec.PersistentVolumeReclaimPolicy)
 	}
 
-	os.RemoveAll(pv.Spec.HostPath.Path)
+	pvDir := filepath.Dir(pv.Spec.HostPath.Path)
```

Comment:
I'm worried this is actually going to delete somebody's data since the provision path is the same for both tests and real clusters.  Let's either leave this out for now, or make the provision path configurable (at least for testing)

Reply:
Done! PTAL

thanks